docs/submit/install-ospool-ap.md

```diff
@@ -4,44 +4,46 @@ title: Installing an Open Science Pool Access Point
 Installing an Open Science Pool Access Point
 ============================================
 
-This document explains how to add a path for user jobs to flow from your local site out to the OSG,
-which in most cases means that the jobs will have far more resources available to run on than locally.
-If your local batch system frequently has many jobs waiting to run for a long time,
-you do not have a local batch system,
-or if you simply want to provide a local entry point for OSG-bound jobs,
-adding a path to OSG may result in less waiting for your users.
+This document explains how to add a path for user jobs to flow from your local site out to the Open Science Pool (OSPool),
+which in most cases means that the jobs will have far more capacity available to run on than locally.
+This is useful if:
+
+- your local batch system frequently has many jobs waiting to run for a long time
+- you do not have a local batch system
+- you want to provide a local entry point for OSPool-bound jobs
```

Comment:
I know you didn't write it but what does this mean? @rynge maybe?

Reply:
Yeah that is poorly written. I think the whole introduction needs a rework. Let's do that separately as well.